yt_dlp/extractor/generic.py

```diff
@@ -2814,6 +2829,18 @@ def _extract_embeds(self, url, webpage, *, urlh=None, info_dict={}):
                 # See https://github.com/ytdl-org/youtube-dl/pull/16735
                 pass
 
+        mobj = re.search(
+            r'<div [^>]*?class="jwplayer"[^>]*data-source=(?P<quote>[\'"])(?P<video_url>[^" >]+)(?P=quote)',
+            webpage)
+        if mobj is not None:
+            return [{
+                '_type': 'video',
+                'url': mobj.group('video_url'),
+                'http_headers': {
```

Comment:
are we sure this is always a direct link (i.e. not a manifest?) if not I would suggest moving this detection to below and set `found` to the urls just in case.